VideoLocker/src/main/java/org/edx/mobile/third_party/iconify/IconDrawable.java

```diff
@@ -203,12 +203,18 @@ public boolean isStateful() {
         return true;
     }
 
+    // TODO: Currently we aren't supporting selectors, when we do, this function will need to be overriden
+    /*
+        The code of the following function is commented because it was messing up with the alpha
+        of the drawable. The resolution is using a ColorFilter. But its out of scope for now.
```

Comment:
Here's how to achieve this effect using a color filter:

```` java
private static final ColorFilter DISABLED_COLOR_FILTER =
        // Half the alpha value
        new PorterDuffColorFilter(0x80FFFFFF, PorterDuff.Mode.MULTIPLY);

@Override
protected boolean onStateChange(int[] state) {
    ColorFilter colorFilter = paint.getColorFilter();
    if (colorFilter == null) {
        if (!isEnabled(state)) {
            setColorFilter(DISABLED_COLOR_FILTER);
            return true;
        }
    } else if (colorFilter == DISABLED_COLOR_FILTER) {
        if (isEnabled(state)) {
            setColorFilter(null);
            return true;
        }
    }
    return false;
}
````

But it's also an arbitrary behaviour that we might wish to customize. So I think we can just remove this solution for the moment, and consider providing a `ColorStateList` implementation later if we need it.